spec/integration/integration_spec_helper.rb

```diff
@@ -64,6 +64,10 @@ def normalize_inspect_output(output)
     gsub!(/(\r\033\[K.*?\r\033\[K).*\r\033\[K(.*)/, "\\1\\2") # strip all progress messages but two
 end
 
+def machinery_host(description)
+  description.sub!(/.*@/, "")
+end
```

Comment:
Could we add some unit tests for this? This would make it easier to see what is happening, because it would be explicitly expressed in the test, and it will also prevent things from breaking in the future.